dev/com.ibm.ws.ejbcontainer.mdb/resources/com/ibm/ws/ejbcontainer/mdb/resources/EJBMDBMessages.nlsprops

```diff
@@ -66,3 +66,11 @@ MDB_ACTIVATION_SPEC_NOT_FOUND_CNTR4015W.useraction=Verify that an activation spe
 MDB_DESTINATION_NOT_FOUND_CNTR4016W=CNTR4016W: The message endpoint for the {0} message-driven bean cannot be activated because the {1} destination is not available. The message endpoint will not receive messages until the destination becomes available.
```

Comment:
Since multiple applications may contain MDBs with the same name; we tend to include text in our messages as follows "the {0} bean in the {1} module of the {2} application", this way it is very clear which MDB by that name we are talking about.  Older messages don't do this; and newer messages don't include this if the context of the message makes it obvious.  For example, if the log will contain a message saying, "Starting Application X", and "Application X started successfully".... and our message will be between those messages, then we don't duplicate the app/module information.
If these messages may appear in a context that does not obviously identify the app/module, then those should be included in the message.  The information may be obtained from bmd._moduleMetaData.ivName, bmd._moduleMetaData.ivAppName

Reply:
I updated the messages to include module and application